python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -146,6 +146,10 @@ def preprocess_ondisk_dataset(
                         graph_feature["name"]
                     ] = node_data
             if graph_feature["domain"] == "edge":
+                if not include_original_edge_id:
+                    dgl_warning(
+                        "Edge feature is stored, but edge IDs are not saved."
+                    )
```

Comment:
Why it is a problem that there are edge attributes in sampling graph while no eid?

Reply:
This check was proposed by Rui, if I have understood him, the reason we raise warning is that edge feature can not be fetched without eids. Loading edge feature with `include_original_edge_id=False` may cause confusion, especially when somebody prepare to use them in the follow-up process.